lib/numbers_and_words/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module NumbersAndWords
-  VERSION = '0.11.12'
+  VERSION = '0.11.11'
```

Comment:
This was correct, next version is 0.11.12 and the current code is incremented for that. 

I do think we should make next version 0.12.0 since we're adding a whole new language. Please change this and the version in CHANGELOG to `0.12.0`.

Reply:
This is still incorrect.